strapi/design-system
# <DatePicker /> component autoComplete issue

### What does it do?

Added `autoComplete="off"` prop to `<DatePicker />` component by default.

### Why is it needed?

Fixes issue with browser's default behaviour.

### How to test it?

Make sure that you have some suggestions from the browser and click on input tag from the `<DatePicker />` component.

### Related issue(s)/PR(s)

https://github.com/strapi/design-system/issues/1797
